talent/job-descriptions/devrel-sf.md

```diff
@@ -0,0 +1,26 @@
+Developer Advocate / Tech Writer
+------------------------------------------
+
+### [Location & Seniority](../location-seniority-section.md)
+
+**Compensation:** ? [stock options](https://github.com/src-d/guide/blob/master/talent/esop.md) for ?
+
+### [Company](../company-section.md)
+
+### [Engineering at source{d}](../engineering-section.md)
+
+### Role
+
+This position is for a Developer Advocate / Tech Writer at source{d}.
```

Comment:
Is this person going to be part of the Developer Relations team? If so, it would be nice to write a small sentence about it, so the candidates will know with who they are going to work with.

Reply:
done